itests/marathon_python.feature

```diff
@@ -1,7 +1,12 @@
 Feature: marathon-python can create and list marathon apps
 
+  Scenario: Metadata can be fetched
+    Given a working marathon instance
+    Then we get the marathon instance's info
+
   Scenario: Trivial apps can be deployed
     Given a working marathon instance
+    Then we get the marathon instance's info
```

Comment:
Can you remove this now? It isn't really related to trivial apps and there is no need for us to exercise this twice